box2d.qbs

```diff
@@ -0,0 +1,30 @@
+import qbs
+
+Project {
+    name: "box2d";
+
+    DynamicLibrary {
+        name: "box2d"
+        files: []
+        Depends { name: "cpp" }
+        Group   { name: "Headers";    files: ["Box2D/*.h", "Box2D/*/*.h", "Box2D/*/*/*.h"]; }
+        Group   { name: "Sources";    files: ["Box2D/*.cpp", "Box2D/*/*.cpp", "Box2D/*/*/*.cpp"]; }
```

Comment:
Why separate the Headers and Sources in different groups? I personally prefer to avoid needlessly complicated file hierarchies so I would merge these into the product itself.